wp-slimstat.php

```
@@ -1245,12 +1255,12 @@ public static function init_options()
     /**
      * Saves a given option in the database
      */
-    public static function update_option($_key = '', $_value = '')
+    public static function update_option($_key = '', $_value = '', bool $_is_network = false)
     {
-        if (!is_network_admin()) {
-            update_option($_key, $_value);
-        } else {
+        if ($_is_network || is_network_admin()) {
             update_site_option($_key, $_value);
```

PHPStan (2.1.46)

[error] Function update_site_option not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(function.notFound)